src/main/java/com/google/gcloud/examples/StorageExample.java

```diff
@@ -336,6 +336,45 @@ public String params() {
     }
   }
 
+  private static class UpdateMetadata extends StorageAction<Tuple<Blob, Map<String, String>>> {
```

Comment:
I agree, this example was mainly used for internal testing and not was a thing I focused on.
I added documentation on the class as well as action and some comments where I thought it would be useful. I didn't want to create specific classes for Data I pass and I don't want to use any third-party in the API level or in the example (so I would rather avoid using jakarta BeanUtil or a like [or use reflection]). I think I found a reasonable compromise.